sql/schema.sql

```diff
@@ -61,3 +61,19 @@ CREATE TABLE label (
   title varchar(100),
   color varchar(100)
 );
+
+-- Create Movie Table
+CREATE TABLE movie (
+  id  integer GENERATED ALWAYS AS IDENTITY,
+  publish_date  date,
+  silent boolean,
+  source_id  int references source(id),
+  PRIMARY KEY(id)
```

Comment:
The movie table looks good, you have included all the necessary columns. You can add one more column to connect the movie table and the item table like an item_id fk in the movie table and you are done 👍

Reply:
Made requested changes to the movie table. Thank you for the correction @rubydevi

Reply:
Great job implementing the required change. 👍